### DISCUSSION TITLE
Ability to hide months in the Calendar

### DISCUSSION THREAD

**Discussion by TimWillebrands** [plouc/nivo#1831] [Ideas]
Hi there,

For a client we're using the calendar component to show the effectiveness of machines over time which looks like this: 
![afbeelding](https://user-images.githubusercontent.com/4950501/141096558-704dfad9-5f6b-4f65-abb5-19ab63d4dbb1.png)
This works great, however we've received complaints about the 'empty months' at the end of the calendar. I've looked into adding a feature to hide months and it doesn't seem too difficult, we'd love to contribute this feature to nivo.

So I wanted to ask whether you'd be interested in such a feature, and if you could you give input on the correct implementation?

I was thinking something along the lines of either: 
`            <Calendar
                {...allTheOtherProps}
                hideMonths={[10, 11]}
            />`
or
`            <Calendar
                {...allTheOtherProps}
                trimEmptyMonths="end" 
            />`
type of trimEmptyMonths could be something like trimEmptyMonths?: "start" | "end" | "both"

Another possibility could be to add borders and such to months in the TimeRange component but that seems a tad more difficult.

I'd love to hear your thoughts

**Comment by TimWillebrands** [comment]
So for good measure we want support to show the calendar like this: 
![afbeelding](https://user-images.githubusercontent.com/4950501/141114199-3a1383c1-cbee-49d0-8515-76ced3ab221f.png)

**Comment by WeismanGitHub** [comment]
I know this is several years old, but I think I found [a solution](https://nivo.rocks/time-range).
